data/orders.ddl

```diff
@@ -0,0 +1,6 @@
+create table orders(
+    order_id INT auto_increment,
+    user_id INT,
+    primary key(order_id),
+    foreign key (user_id) references users(user_id)
```

Comment:
Do you see why you can just use "id" as the column name in the users table?  Because in this table you call it user_id and the foreign key line would look like
foreign key (user_id) references users(id)

Which is readable and makes sense.